cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,44 @@
+import PageObject from '../PageObject';
+
+class settingsPageObject extends PageObject {
```

Comment:
Class name should be in PascalCase. Rename 'settingsPageObject' to 'SettingsPageObject'